app/views/courses/_course_overview.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="objectives">
+  <h4>Learn</h4>
+
+  <ul class="objectives-list">
+    <% course.you_learn.each do |objective| %>
+      <li><%= objective %></li>
+    <% end %>
+  </ul>
+</div> <!-- /.objectives -->
+
+<div class="projects">
+  <h4>Projects</h4>
+
+  <ul class="projects-list">
+    <% course.you_build.each do |project| %>
+      <li><%= project %></li>
+    <% end %>
+  </ul>
+
+  <%= render 'see_more_link', course: course %>
```

Comment:
@KevinMulhern In the current design(production version), we display the 'See More' links twice. Do we _really_need it to be displayed twice?

If not, I will move the partial `_see_more_link` to `_course_overview` partial.